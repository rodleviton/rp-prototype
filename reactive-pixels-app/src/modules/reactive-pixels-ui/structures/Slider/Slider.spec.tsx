import * as React from "react";
import { render } from "react-testing-library";
import WithRouter from "../../utils/test/WithRouter";
import WithTheme from "../../utils/test/WithTheme";
import { Slider } from "./";

describe("<Slider />", () => {
  it("should render <Slider /> component", () => {
    const { container } = render(
      <WithRouter>
        <WithTheme>
          <Slider />
        </WithTheme>
      </WithRouter>
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
