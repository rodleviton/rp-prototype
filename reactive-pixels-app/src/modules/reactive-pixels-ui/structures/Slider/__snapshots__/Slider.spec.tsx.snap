// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Slider /> should render <Slider /> component 1`] = `
<div
  class="css-0 css-lkyta4 sc-bwzfXH jVnbds sc-bdVaJa bDWFJH"
/>
`;
